### PR TITLE
SGD8-2568: Add teaser sq double + fix grid

### PR DESCRIPTION
**Pull request by SvenVdv**
## Description

- Fix some grid legacy issues
- Add teaser double template + fix square template

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: templates/custom/nodes/node--news--teaser-double.html.twig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% extends 'node--teaser.html.twig' %}
```
Comment (lennartvava) [inline templates/custom/nodes/node--news--teaser-double.html.twig]: Klopt de template naam nog met de view mode teaser-wide?

Comment (SvenVdv) [reply templates/custom/nodes/node--news--teaser-double.html.twig]: Ja, klopt! Heb het aangepast :)

File: templates/custom/nodes/node--news--teaser.html.twig [hunk 2]
```diff
@@ -12,5 +12,18 @@
 
   {% set image_ratio = '1:1' %}
   {# twigcs use-var image_ratio #}
-  {{ parent() }}
+  {% if node.hasField('field_teaser_image') %}
+    {% if image_ratio is not defined %}
+      {% set image_ratio = '8:5' %}
```
Comment (lennartvava) [inline templates/custom/nodes/node--news--teaser.html.twig]: Is deze niet altijd set op regel 13?

Comment (SvenVdv) [reply templates/custom/nodes/node--news--teaser.html.twig]: Klopt. Aangepast zodat het nu 1:1 is standaard

**Review by lennartvava — APPROVED** [review]

**Pull request merged**